main.go

```diff
@@ -57,21 +77,30 @@ func main() {
 	fmt.Printf("Found %d vulnerabilities \n", len(vs))
 	highSevNumber := len(store["High"]) + len(store["Critical"]) + len(store["Defcon1"])
 
-	iteratePriorities(func(sev string) {
+	iteratePriorities(clairOutput, func(sev string) {
 		for _, v := range store[sev] {
 			fmt.Printf("%s: [%s] \n%s\n%s\n", v.Name, v.Severity, v.Description, v.Link)
 			fmt.Println("-----------------------------------------")
 		}
 	})
-	iteratePriorities(func(sev string) { fmt.Printf("%s: %d\n", sev, len(store[sev])) })
+	iteratePriorities(priorities[0], func(sev string) { fmt.Printf("%s: %d\n", sev, len(store[sev])) })
 
 	if highSevNumber > threshold {
 		os.Exit(1)
 	}
 }
 
-func iteratePriorities(f func(sev string)) {
+func iteratePriorities(output string, f func(sev string)) {
+	filtered := true
```

Comment:
Do you think the `filtered` is not enough readable? Not sure whether the following explain is acceptable.
In the case of iteration over a subslice of `priorities`, there will be start and end index. Both for priorities before the start index and after the end index, `filtered` is true as they will be filtered. Only for priorities between the start index and the end index, `filtered` is false as they will be outputted.